### ISSUE TITLE
XFA browser not showing correct text

### ISSUE THREAD

**Issue by brendandahl** [brendandahl/pdf.js.utils#8]
https://bug1722003.bmoattachments.org/attachment.cgi?id=9232818
```
if you open this pdf: https://bug1722003.bmoattachments.org/attachment.cgi?id=9232818
and search for "Text2" string
you'll see something like <exData contentType="text/html"> and the stuff under this node is almost empty (only white spaces with some spans around)
and if I uncompress it using qpdf I can see the text under this node
"Please read the ...."
```

**Comment by brendandahl** [comment]
If I download the raw section of the template the expected text is there. I'm guessing there's something wrong with the XML prettifier.

**Comment by brendandahl** [comment]
@calixte
Fixed by 95d9dd565063f74bfba7bbe23609e8cb41f07766
